java/buildconf/ivy/obs-maven-config.yaml

```diff
@@ -26,7 +23,7 @@ artifacts:
     repository: Uyuni_Other
   - artifact: cglib
     jar: cglib.jar
-    repository: Leap15.1
+    repository: Leap
```

Comment:
Not saying this is wrong, but I don't understand it: why does it change from `Leap15.1` to `Leap`

Reply:
Leap is the alias for the latest Leap repo which is 15.2 in this case. If you want to rename it, feel free to do it :)

Reply:
No, now that I you told, it makes sense. Most likely you already documented it for `obs-to-maven` and I am just not familiar with it :-)

Reply:
The standard Leap is actually 15.2, see [here](https://github.com/uyuni-project/uyuni/blob/master/java/buildconf/ivy/obs-maven-config.yaml#L2-L4). This way we only need to change it in one place when 15.3 comes.